tests/ui/traits/object/crash-due-to-projections-modulo-norm.rs

```diff
@@ -1,9 +1,10 @@
-//@ known-bug: rust-lang/rust#126944
+//@ check-pass
+
```

Comment:
```suggestion

// Regression test for #126944.
```
And "fixed #126944" in the PR descr.